templates/zerver/help/restrict-private-messages.md

```diff
@@ -2,10 +2,11 @@
 
 {!admin-only.md!}
 
-By default, any user can send private messages.  Organization
-administrators can change who is allowed to use private messages.
+In Zulip, any user can exchange private messages with other users (and
+[bots](/help/bots-and-integrations)). Organization administrators can
```

Comment:
Let's change "any user" -> "users" to make it feel less inconsistent with the next sentence.

Also, it's not *quite* accurate, in that you can also send PMs to yourself (not just other users). Maybe that's OK?

Reply:
Hmm... good point. Updating so that it reads:

> In Zulip, a user can exchange private messages with other users, [bots](https://zulip.com/help/bots-and-integrations) and themself. Organization administrators can configure who is allowed to use private messages.